static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx

```diff
@@ -24,6 +24,7 @@ type Props = ModalRenderProps & {
   onDelete: () => void;
   orgSlug: Organization['slug'];
   projectSlug: Project['slug'];
+  trackDownload: () => void;
```

Comment:
I feel like we could just call this `onDownload` in case we ever want to use it for something else in the future